src/pages/ContentfulPage.vue

```diff
@@ -111,13 +113,26 @@ const getPageFrameFromType = type => {
 	}
 };
 
+// The components that return kv-tailwind will get wrapped in that class,
+// they have already been migrated to use tailwind CSS. Legacy components
+// should return empty string so they are not wrapped in the kv-tailwind class
+// once all components use tailwind, we can add the 'kv-tailwind' to the
+// :is="pageFrame" component on this page
+const getWrapperClassFromType = type => {
+	switch (type) {
+		case 'landingPageHero':
```

Comment:
Not sure what to name this component. Suggestions?

Reply:
Maybe "heroWithCarousel" ...